pages/bridge.tsx

```diff
@@ -213,14 +223,17 @@ const Bridge = () => {
                     type="button"
                     className="mt-5 mx-auto flex flex-row items-center rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 disabled:bg-gray-300 disabled:cursor-not-allowed"
                     onClick={handleBridge}
-                    disabled={bridgeTarget === BridgeTarget.L2 ? !bridgeToL2 : !bridgeToL1}
+                    disabled={
+                      bridgeTarget === BridgeTarget.L2 ? bridgeToL2IsError : bridgeToL1IsError
```

Comment:
What inspired this change?
can we gray out the button as disabled while the spinner is showing? I think that the earlier version of this line may accomplish that.

Reply:
umm, tbh I'm not sure anymore after the `easyWrite` changes, but I had to add extra params to the earlier version to gray it out. Check commit: [5a75f01](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/58/commits/5a75f019426ca142d2bd83a49a7bb42e3e353c87)